src/main/kotlin/org/rust/ide/colors/RustColors.kt

```diff
@@ -9,11 +9,11 @@ import com.intellij.openapi.editor.DefaultLanguageHighlighterColors as Default
  */
 enum class RustColor(humanName: String, val default: TextAttributesKey) {
     IDENTIFIER            ("Identifier",                  Default.IDENTIFIER),
-    FUNCTION_DECLARATION  ("Function declaration",        Default.FUNCTION_DECLARATION),
-    INSTANCE_METHOD       ("Instance method declaration", Default.INSTANCE_METHOD),
-    STATIC_METHOD         ("Static method declaration",   Default.STATIC_METHOD),
-    PARAMETER             ("Function parameter",          Default.PARAMETER),
-    SELF_PARAMETER        ("Self parameter",              Default.KEYWORD),
+    FUNCTION_DECLARATION  ("Function",                    Default.FUNCTION_DECLARATION),
+    INSTANCE_METHOD       ("Method",                      Default.INSTANCE_METHOD),
+    STATIC_METHOD         ("Associated function",         Default.STATIC_METHOD),
+    PARAMETER             ("Function argument",           Default.PARAMETER),
```

Comment:
Hm, interesting, I never thought of arguments/parameters in this way.

It seems that the book (and the error index as well) is quite definite about using "argument" when talking [about functions](http://rust-lang.github.io/book/ch03-03-how-functions-work.html) and "parameter" when talking [about generic parameters](https://doc.rust-lang.org/book/generics.html) with not a single exception, while `rustc` emits messages that contain both terms and don't follow any rules about them.

I'm not an adherent of any of these terms, but the "parameter" seems to contradict even the very next "Self argument" (though named SELF_PARAMETER internally) element. Should it be "Self parameter"?